models/platform-limits/src/general/generator.ts

```diff
@@ -0,0 +1,33 @@
+import { Generator } from '@commercetools-test-data/core';
+import { TProjectCustomLimitsProjection } from './types';
+
+/**
+ * customers -  reference to customersPlatformLimits ,
```

Comment:
These comments might be confusing as "reference" is also a model type.
Perhaps we can omit them as the `types.ts` file already has this information.

Reply:
Done https://github.com/commercetools/test-data/pull/657/commits/6bfab6455c8b504b50e7e12d8b8c3e1da173dcef#diff-c5cb7d76dd373d32885396efb72c25d87c7d548a6671dba26273537a306cdfa4